tools/xcms_xcmsset/abims_xcms_xcmsSet.xml

```diff
@@ -183,6 +187,12 @@
     <outputs>
         <data name="xsetRData" format="rdata.xcms.findchrompeaks" label="${image.name.rsplit('.',1)[0]}.xset.RData" from_work_dir="xcmsSet.RData" />
         <data name="log" format="txt" label="${image.name.rsplit('.',1)[0]}.xset.log.txt" from_work_dir="log.txt" />
+		<data name="peaklist_out1" format="tabular" label="${image.name[:-6]}.chromPeak_table.tsv" from_work_dir="chromPeak_table.tsv" >
+            <filter>methods['method'] == 'CentWave' and (methods['CentWaveAdv']['peaklist'])</filter>
+        </data>
+		<data name="peaklist_out2" format="tabular" label="${image.name[:-6]}.chromPeak_table.tsv" from_work_dir="chromPeak_table.tsv" >
+            <filter>methods['method'] == 'MatchedFilter' and (methods['MatchedFilterAdv']['peaklist'])</filter>
+        </data>
```

Comment:
There is an indentation issue here.
Can you use 4 spaces instead of tabulation?